x-pack/test/functional/apps/infra/node_details.ts

```diff
@@ -623,6 +640,45 @@ export default ({ getPageObjects, getService }: FtrProviderContext) => {
             });
           });
         });
+
+        describe('Callouts', () => {
+          describe('Legacy alert metric callout', () => {
+            [{ metric: 'cpu' }, { metric: 'rx' }, { metric: 'tx' }].forEach(({ metric }) => {
+              it(`Should show for: ${metric}`, async () => {
+                await navigateToNodeDetails('Jennys-MBP.fritz.box', 'host', {
+                  name: 'Jennys-MBP.fritz.box',
+                  incomingAlertMetric: metric,
+                });
+                await pageObjects.header.waitUntilLoadingHasFinished();
+
+                await retry.try(async () => {
+                  expect(await pageObjects.assetDetails.legacyMetricAlertCalloutExists()).to.be(
+                    true
+                  );
+                });
+              });
+            });
+
+            [{ metric: 'cpuTotal' }, { metric: 'rxV2' }, { metric: 'txV2' }].forEach(
```

Comment:
I think we changed `cpuTotal` to `cpuV2`, right?

Reply:
good catch! will fix it. thanks.